DFC.Digital/DFC.Digital.Service.CourseSearch/Utility/CourseSearchConstants.cs

```diff
@@ -0,0 +1,75 @@
+﻿namespace DFC.Digital.Service.CourseSearchProvider
+{
+    public static class CourseSearchConstants
+    {
+        //https://coursedirectoryproviderportal.org.uk/Content/Files/Help/Course%20Directory%20API.pdf
+        //6.5. Attendance Modes
+        //Code Description
+        //AM1 Location / campus
+        //AM2 Face-to-face(non-campus)
+        //AM3 Work-based
+        //AM4 Mixed Mode
+        //AM5 Distance with attendance
+        //AM6 Distance without attendance
+        //AM7 Online without attendance
+        //AM8 Online with attendance
+        //AM9 Not known
+        public const string AllAttendanceModes = "AM1,AM2,AM3,AM4,AM5,AM6,AM7,AM8,AM9";
```

Comment:
if this will never get used as string and will always be split to arrays, then this should not be a const string, instead use IEnumerable<string> e.g.
```suggestion
        public static IEnumerable<string> AllAttendanceModes 
        {
                get 
                { 
                      return new string[]
                      {
                          "AM1", //Location / campus
                          "AM2", //Face-to-face(non-campus)
                          "AM3", //Work-based
                          "AM4", //Mixed Mode
                          "AM5", //Distance with attendance
                          "AM6", //Distance without attendance
                          "AM7", //Online without attendance
                          "AM8", //Online with attendance
                          "AM9" //Not known
                      };
                }
         }
```